app/players/player.py

```diff
@@ -0,0 +1,15 @@
+from abc import ABC, abstractmethod
+
+
+class Player(ABC):
+
+    def __init__(self, nickname: str) -> None:
+        self.nickname = nickname
+
+    @abstractmethod
+    def get_rating(self) -> None:
+        pass

```

Comment:
It accepts int. You must specify type which you expect from all child classes
The same in player_info

Reply:
accepts? does not it returns int so?

Reply:
so returns 😁